modules/sso/class.jetpack-sso-notices.php

```diff
@@ -25,8 +27,8 @@ public static function error_msg_enable_two_step( $message ) {
 				),
 				array(  'a' => array( 'href' => array() ) )
 			),
-			'https://wordpress.com/me/security/two-step',
-			'https://support.wordpress.com/security/two-step-authentication/'
+			Redirect::get_url( 'calypso-me-security-2fa' ),
+			Redirect::get_url( 'wpcom-support-security-2fa' )
```

Comment:
Should we use Calypso here? I also wonder if we should use `two-step-authentication` here as well, so it's easier to remember?

Reply:
I don't think we should use calypso here because the link goes to wordpress.com/support.

I'm not expecting no one to memorize these slugs. Also, in calypso the url is jus `two-step`, while in wp.com support is `two-step-authentication`

What do you think?

Reply:
> in calypso the url is jus two-step, while in wp.com support is two-step-authentication

What do you think about using the same naming here as well?
`calypso-me-security-two-step`
`wpcom-support-security-two-step-authentication`

Reply:
Makes sense. Done in 72b8023ec443dc7b3a89e610fd028bdee0db1671